### PR TITLE
script for copying a database from pod to local

### PR DESCRIPTION
**Pull request by Sybrand**
Script for copying database from a pod in openshift, to you local machine.

Note: There may be all kinds of issues re. how you connect to postgress locally, so the restore part of the script may fail - but at least you'll have the database locally, and should be able to figure out a restore from there onwards.

### CODE REVIEW

File: scripts/copy_db_from_pod_to_local.sh [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+# Usage:
+#
+#   PROJECT=OPENSHIFT_PROJECT_NAME POD=OPENSHIFT_POD DATABASE=POSTGRESQL_DATABASE ${THIS_FILE}
+#
+#   Assumes you have openshift command line tools installed and are logged in.
+#
+# Examples:
+#   
+#   Copy database from a pod in production.
+#   PROJECT=auzhsi-prod POD=patroni-wps-api-prod-2 DATABASE=wps-api-prod ./restore_db.sh
```
Comment (Kyubinhan) [inline scripts/copy_db_from_pod_to_local.sh:16]: Did you forget to rename `restore_db.sh`?

File: scripts/copy_db_from_pod_to_local.sh [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+# unzip it locally
+gunzip "${FILENAME}.gz"
+
+####################
+# Restore it locally
+####################
+
+# now do the restore (will drop and re-create users etc.)
+# will give you errors the first time you run it, since there's nothing to drop.
+RESTORE_COMMAND="pg_restore -d ${DATABASE} -U postgres -c ${FILENAME}"
```
Comment (andrea-williams) [inline scripts/copy_db_from_pod_to_local.sh:96]: I was looking at https://www.postgresql.org/docs/current/app-pgrestore.html to try to understand what the pg_restore command does, and it looks like if you add `--clean --if-exists` to the command, you won't get an error message about having nothing to drop?

File: scripts/copy_db_from_pod_to_local.sh (hunk 1 above)
Comment (Sybrand) [reply scripts/copy_db_from_pod_to_local.sh]: I did indeed.

File: scripts/copy_db_from_pod_to_local.sh (hunk 2 above)
Comment (Sybrand) [reply scripts/copy_db_from_pod_to_local.sh]: Thanks Andrea - I've change the comments to reflect that, and added another example of how to restore.

**Pull request merged**